package.json

```diff
@@ -35,10 +35,10 @@
     "eslint": "^1.10.3",
     "mocha": "^2.4.5",
     "publish-please": "^2.2.0"
-  },
-  "dependencies": {
     "@types/error-stack-parser": "^1.3.18",
     "@types/lodash": "^4.14.72",
+  },
```

Comment:
This is an invalid JSON, so build fails: comma required after the `publish-please` dependency.

Reply:
@inikulin, I have created a follow-up PR here: https://github.com/inikulin/callsite-record/pull/26